### PR TITLE
Created FilesystemModelProvider and refactored tests

### PR DESCRIPTION
**Pull request by marcin-kolda**
Tests no longer highly depends on number of table/view files

### CODE REVIEW

File: tests/filesystem_model_test.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import unittest
+
+from gcp_census.model.filesystem_model_provider import FilesystemModelProvider
+
+
+class TestFilesystemModelProvider(unittest.TestCase):
+
+    def test_should_return_at_least_3_groups(self):
```
Comment (przemyslaw-jasinski) [inline tests/filesystem_model_test.py:8]: Not sure about this, I would like to discuss. If you change bq_schemas conf then you need to fix tests.

File: gcp_census/model/filesystem_model_provider.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+from gcp_census.model.view import View
+
+
+class FilesystemModelProvider(object):
+
+    def __init__(self, model_directory):
+        self.model_directory = model_directory
+
+    def list_tables(self):
+        for table in self.__list_files('.json'):
```
Comment (przemyslaw-jasinski) [inline gcp_census/model/filesystem_model_provider.py:14]: Why do you assume that files with '.json' extension are tables and with '.sql' are views?

**Review by przemyslaw-jasinski — CHANGES_REQUESTED** [review]
I'd like to discuss why certain things were made this way.

File: tests/filesystem_model_test.py (hunk 1 above)
Comment (marcin-kolda) [reply tests/filesystem_model_test.py:8]: Previously all changes in schema number required to update the tests, now we are only testing the provider to return required minimum

File: gcp_census/model/filesystem_model_provider.py (hunk 2 above)
Comment (marcin-kolda) [reply gcp_census/model/filesystem_model_provider.py:14]: That's our convention, as for views we don't need to specify any metadata.

**Review by przemyslaw-jasinski — APPROVED** [review]

**Review by przemyslaw-jasinski — APPROVED** [review]

**Review by przemyslaw-jasinski — APPROVED** [review]

**Review by przemyslaw-jasinski — APPROVED** [review]

**Pull request merged**